Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -1978,19 +2139,67 @@ private void SwitchControlSchemeInternal(ref InputControlScheme controlScheme, p
             }
         }
 
+        /// <summary>
+        /// An event associated with an PlayerInput action.
+        /// </summary>
+        /// <remarks>
+        /// Represents an event invoked in response to actions being triggered.
+        ///
+        /// Contains the ID and name of the <see cref="InputAction"/> being triggered and the associated <see cref="UnityAction"/> to handle the action response.
+        ///
+        /// The list of action events is specified in <see cref="PlayerInput"/> Editor UI based on the selected <see cref="InputActionAsset"/>.
+        /// The individual action callbacks are then specified in a <see cref="MonoBehaviour"/>.
+        ///
+        /// <example>
+        /// <code>
+        /// public class MyPlayerScript : MonoBehaviour
+        /// {
+        ///     void OnFireEvent(InputAction.CallbackContext context)
+        ///     {
+        ///         // Handle fire event
+        ///     }
+        /// }
+        /// </code>
+        /// </example>
+        /// </remarks>
+        /// <seealso cref="PlayerInput.actionEvents"/>
```

Comment:
Should we add a seals here for InputAction.CallbackContext as well since that what the sample shows?

Reply:
Adding in next commit